test/unit/gateways/decidir_test.rb

```diff
@@ -38,6 +38,9 @@ def setup
         amount: 1500
       }
     ]
+    @customer_options = {
```

Comment:
Did this get orphaned? Is it used anywhere? 🧐

Reply:
Lol, yes. Was going to add it like this but then decided it was unnecessary and forgot to remove it. Thanks!

Reply:
Fixed